crates/bevy_render/src/camera/projection.rs

```diff
@@ -53,6 +53,9 @@ impl<T: CameraProjection + Component + GetTypeRegistration> Plugin for CameraPro
 /// to recompute the camera projection matrix of the [`Camera`] component attached to
 /// the same entity as the component implementing this trait.
 ///
+/// Components implementing this trait include: [`Projection`], [`PerspectiveProjection`]
+/// and [`OrthographicProjection`].
+///
```

Comment:
this is hard to maintain doc, and already done by docs.rs, see https://docs.rs/bevy/latest/bevy/render/camera/trait.CameraProjection.html#implementors

Reply:
Removed in https://github.com/bevyengine/bevy/pull/9136/commits/dee7a95f4e34e2534c35ebcbf8a292c764dd5519